deepreg/config/parser.py

```diff
@@ -80,15 +80,12 @@ def config_sanity_check(config: dict) -> dict:
     :param config: entire config.
     """
 
-    # check data
-    data_config = config["dataset"]
-
     # back compatibility support
     config = parse_v011(config)
 
     # check model
     if config["train"]["method"] == "conditional":
-        if data_config["labeled"] is False:  # unlabeled
+        if config["dataset"]["train"]["labeled"] is False:  # unlabeled
             raise ValueError(
                 "For conditional model, data have to be labeled, got unlabeled data."
```

Comment:
just a note here: for conditional seg. only moving images "have to" be labelled for val and test splits.

Reply:
Good point! But that's yet another level of complexity :(